### ISSUE TITLE
Self-responses, maybe based on retweets?

### ISSUE THREAD

**Issue by duetosymmetry** [persones/yogimark#1]
Noticed [this interaction](https://twitter.com/YogiMarkov/status/895676118123945988):

<img width="654" alt="screen shot 2017-08-10 at 11 16 47 am" src="https://user-images.githubusercontent.com/679461/29185273-96d75ffe-7dbd-11e7-8f9c-1eec1a01a085.png">

**Comment by persones** [comment]
Yeah, it seems it can't tell the difference between a tweet where it's mentioned and a retweet. I'll look into that.
